JAVA/Webscrapper/README.md

```diff
@@ -0,0 +1,51 @@
+# Webscrapper
+A GUI based webapplication that shows all the the live covid details reguarding any country.
+
+
+***Web Scrapper*** 
+
+**Technology i have used are Java and Jsoup library.**
+
+What the project does is it will find the covid 
+cases in the countries you will be searching for.
+For the file first  you need to have java for that you will be needing java development kit.
+Then you will be needing a editor or compiler like Vscode , IntelijIdea and e.t.c
+
+Then you can redirect to the folder choose java and  then
+You will find their a Webscrapper file open it using some kind of text editor and run it.
+
+
+This is how it looks before you enter something.
+
+![pr](https://user-images.githubusercontent.com/86187166/156936044-b5156174-3716-4939-8dae-675e30e36270.PNG)
+
+Here is what it looks after you enter it.
+
+![pr1](https://user-images.githubusercontent.com/86187166/156936063-cd635554-eb12-492a-ac1a-65886495732e.PNG)
+
+Here is a tutorial how to open this
+
+**1.Git clone the repository to your local machine**
+
+**2.Open the folder with any editor you use.**
+
+**3.Go to Webscrapper>src>main>java>com.tridib>App**.
+
+**4.Run the App.java file.**
```

Comment:
@Tridib11 mentione this in these manner

Add this two sections (install dependencies and Quick Start) in [Readme.md](http://readme.md/) file in these particlular manner : 
<img width="1100" alt="Screenshot 2022-03-03 at 1 02 07 PM" src="https://user-images.githubusercontent.com/78725162/158399068-9c84a79b-17d7-47cc-b1ec-02ee536cfcd2.png">
<br />
**Quick Start** (how to run in local)
<hr />
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/158399122-5fcc235c-b2af-4607-ac48-5e97b9481f4b.png">